.github/workflows/update_model_listing.yml

```diff
@@ -0,0 +1,147 @@
+name: Update Pretrained Model Listing
+on:
+  workflow_dispatch:
+  
+jobs:
+  update-model-listing:
+    runs-on: 'ubuntu-latest'
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+    environment: opensearch-py-ml-cicd-env
+    env:
+      bucket_model_listing_file_path: ml-models/model_listing/pre_trained_models.json
+      repo_model_listing_path: ./utils/model_uploader/model_listing/pretrained_model_listing.json
+      path_prefixes: "ml-models/huggingface/" 
+      # If you want to the model listing to include more folders you just need to modify path_prefixes
```

Comment:
How about this:


To expand the model listing to encompass additional folders, simply adjust the path_prefixes as indicated below: "ml-models/first_folder/ ml-models/second_folder/ ml-models/third_folder/" (Separate each folder with a space)